integrations/visual_studio.rst

```diff
@@ -4,114 +4,79 @@
 |visual_logo| Visual Studio
 ===========================
 
-Conan can be integrated with **Visual Studio** in two different ways:
+Microsoft Visual Studio is one of the most used IDEs to develop C++ applications and Conan has
+many utilities to work with it. This section serves as a reference to briefly describe some
+of them and link to their own sections, but it will also contain documentation related to
+some utilities that are totally dedicated to Visual Studio itself.
 
-- Using the ``cmake`` generator to create a *conanbuildinfo.cmake* file.
-- Using the ``visual_studio`` generator to create a *conanbuildinfo.props* file.
+There are two different ways that can be used to integrate Conan into Visual Studio, always
+requiring the generators provided by Conan:
 
-With CMake
-----------
+.. toctree::
+   :maxdepth: 2
 
-Use the ``cmake`` generator or ``cmake_multi`` if you are using CMake to machine-generate your Visual Studio projects.
+   Using Conan provided CMake generators <visual_studio/cmake_generators>
+   Using Conan provided Visual Studio generators <visual_studio/visual_studio_generators>
 
-Check the :ref:`generators` section to read about the ``cmake`` generator.
-Check the official `CMake docs`_ to find out more about generating Visual Studio projects with CMake.
 
-However, beware of some current CMake limitations, such as not dealing well with find-packages, because CMake doesn't know how to handle finding both debug and release packages.
+Conan Extension for Visual Studio
+---------------------------------
 
-.. note::
+Thanks to the invaluable help of our community we manage to develop and maintain a free extension
+for Visual Studio in the Microsoft Marketplace, it is called Conan Extension for Visual
+Studio and provide and integration with Conan using the some of the :ref:`Visual Studio
+generators <visualstudio_generator>`.
 
-    If you want to use the Visual Studio 2017 + CMake integration, :ref:`check this how-to<visual2017_cmake_howto>`
+.. image:: ../images/visual_studio/marketplace-header.png
+   :width: 90%
+   :alt: Conan Extension for Visual Studio in the Microsoft marketplace
 
-With *visual_studio* generator
-------------------------------
+You can install it into your IDE using the **Extensions manager** and start using it right away,
+this extension will look for a *conanfile.txt* in the solution and retrieve the requirements
```

Comment:
```suggestion
this extension will look for a *conanfile.txt* (or *conanfile.py*) in the solution and retrieve the requirements
```

Reply:
I think it is better the other way around, as the *conanfile.py* has preference over the *conanfile.txt*